src/styles/tabs.js

```diff
@@ -32,7 +40,7 @@ export default ({ tabBarComponent, tabBarPosition, showLabel, showIcon, tabWidth
       flex: 1,
     },
     style: {
-      backgroundColor: 'transparent',
+      backgroundColor: theme === 'night' ? '#212D3B' : 'white',
```

Comment:
Try to reuse `#212D3B` from https://github.com/zulip/zulip-mobile/blob/bd915517f3f45d49d0e0174bc297409724331186/src/styles/themeDark.js#L4. So in later stage if we change, it will be easy & less error prone.

Reply:
Done. 👍